linetools/isgm/io.py

```diff
@@ -179,3 +181,100 @@ def write_joebvp_from_components(comp_list, specfile, outfile):
         s = comp.repr_joebvp(specfile, flags=flags, b_default=b_val)  # still, b values from abslines take precedence if they exist
         f.write(s)
     f.close()
+
+
+def read_igmg_to_components(igmguesses_json):
+    """Creates a list of AbsComponenbts from a igmguesses_json file
+
+    Parameters
+    ----------
+    igmguesses_json : str
+        Name of the json file generated by IGMGuesses
+
+    Returns
+    -------
+    comps : list
+        A list of AbsComponents
+
+    """
+    # Read the JSON file
+    with open(igmguesses_json) as data_file:
+        igmg_dict = json.load(data_file)
+    # Components
+    comp_list = []
+    icoord = SkyCoord(ra=igmg_dict['meta']['RA'], dec=igmg_dict['meta']['DEC'], unit='deg')
+    for ii, key in enumerate(igmg_dict['cmps'].keys()):
+        # import pdb;pdb.set_trace()
+        comp_dict = igmg_dict['cmps'][key]
+        comp_dict['flag_N'] = 1
+        comp_dict['logN'] = comp_dict['Nfit']
+        comp_dict['sig_logN'] = -1
+        try:
+            comp = AbsComponent.from_dict(comp_dict, coord=icoord, chk_sep=False, chk_data=False, chk_vel=False, linelist="ISM")
+        except:  # is this a H2 line?
+            comp = AbsComponent.from_dict(comp_dict, coord=icoord, chk_sep=False, chk_data=False, chk_vel=False, linelist='H2')
+        comp_list += [comp]
+    # sort list by redshift
+    redshifts = [comp.zcomp for comp in comp_list]
+    inds = np.argsort(redshifts)
+    comp_sorted = list(np.array(comp_list)[inds])
+    #return
+    return comp_sorted
+
+
+def write_igmg_from_components(comp_list, specfile, fwhm, outfile='IGM_model.json'):
```

Comment:
Again, this should be in pyigm.  And 
make it work with IGMSightline which
I don't think has a write() method yet.